### ISSUE TITLE
Nuxt Axios interceptors not working when Axios plugin is in Auth plugin option.

### ISSUE THREAD

**Issue by alucardu** [nuxt-community/auth-module#1026]
### Version
auth: 4.9.1
axios: 5.12.2
nuxt: 2.14.6

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [ X ] universal
 - [ ] spa

### Nuxt configuration
```
    auth: {
      redirect: {
        login: '/',
        logout: '/',
        callback: '/',
        home: '/'
      },
      plugins: [
        { src: '~plugins/axios' }
      ],
      strategies: {
        facebook: {
          client_id: '**',
          scope: ['public_profile', 'email']
        },
      }
    },
```
-->

## Reproduction
> :warning: without a minimal reproduction we won't be able to look into your issue

https://codesandbox.io/s/lively-tree-b84o6?

<!--

#### What is expected?

When adding the axios.js plugin to the auth plugin option the axios interceptors should catch the request/response.

#### What is actually happening?

When the axios.js plugin is added to the global nuxt plugin option the axios intercepters catch the request/response. When the plugin is added to the auth plugin option the axios interceptors do not get triggered. 

#### Steps to reproduce

1:
Go to the sandbox: https://codesandbox.io/s/lively-tree-b84o6?
Open console
Login using the Facebook button
See that there is no response/request logging:

![image](https://user-images.githubusercontent.com/779921/105647381-89d53c00-5ea5-11eb-8ff0-73c9e49298ae.png)

2:
In nuxt.config.js comment out line: 33 (auth plugin)
Uncomment line: 44 (nuxt global plugin)
Open console
Login using the Facebook button
See that there is response/request logging:

![image](https://user-images.githubusercontent.com/779921/105647356-5f837e80-5ea5-11eb-96f8-4c98415a016f.png)

## Checklist
* [ ]  I have tested with the latest Nuxt version and the issue still occurs
* [ ]  I have tested with the latest module version and the issue still occurs
* [ X ]  I have searched the issue tracker and this issue hasn't been reported yet

Due to reasons I cannot upgrade nuxt, auth and axios.

### What is expected?

Axios interceptors should work when added to the auth plugin option

**Comment by bmulholland** [comment]
Thanks for reporting! We're focused on the v5 branch now - is the issue present if you use `@nuxtjs/auth-next`?

**Comment by alucardu** [comment]
@bmulholland yes, I've forked the sandbox > https://codesandbox.io/s/condescending-dawn-54w9d and updated the nuxt auth package to nuxt auth-next and it's the same issue. The console log in the axios plugin is not reached when the plugin is added to the auth plugins, but is reached when added to the global nuxt plugins option. One thing, in the forked sandbox you might need to reload the page to see the console log after you've logged in to facebook (when you added the axios plugin to the nuxt global plugin option).

**Comment by jaxenvisuals** [comment]
This is still an issue. Is there a solution now?

**Comment by rvmourik** [comment]
I am also experiencing the same issue, which is leading to issues processing the potential errors that are catched in the onError call.

Is there a possible work around for the time being?

**Comment by bmulholland** [comment]
Hey all - apologies, I simply don't have bandwidth to look into this right now. Any debugging and/or fixes you can come up with can be express reviewed, though.

**Comment by pawmanaloto** [comment]
Any fix on this? Also experiencing this issue.

**Comment by bmulholland** [comment]
@pawmanaloto Your investigation and PR are welcomed :)

**Comment by bmulholland** [comment]
Axios was recently upgraded -- does that help with this issue?
